NuGet.config

```diff
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="re-motion Debug-Builds" value="https://nuget.re-motion.org/nuget/re-motion-debug/" />
-    <add key="re-motion Infrastructure" value="https://nuget.re-motion.org/nuget/re-motion-infrastructure/" />
```

Comment:
As discussed, we need those to restore the packages in the Build project. Projects with a `packages.config` file won't be affected by central package management but they will be affected if we change the packages sources here